jenkins.el

```diff
@@ -114,7 +114,11 @@
 
 (defun get-jenkins-url ()
   "This function is for backward compatibility."
-  (or jenkins-url jenkins-hostname))
+  (let ((url (or jenkins-url jenkins-hostname)))
```

Comment:
:) I thought about that too!

It feels like a slight abuse of that function. `file-name-as-directory` downcases on Windows and calls `file-name-handler` according to `file-name-handler-alist`. Those could break legitimate URLs, e.g. `http://foo/BAR` is not the same as `http://foo/bar`.